util/util.go

```diff
@@ -424,6 +424,20 @@ func ActivateLibStorage(
 		errs <-chan error
 	)
 
+	// generate cert/key pair if necessary
+	certPath := config.GetString(apitypes.ConfigTLSCertFile)
+	keyPath := config.GetString(apitypes.ConfigTLSKeyFile)
+	host := config.GetString(apitypes.ConfigHost)
+
+	ctx.WithFields(log.Fields{
+		"host":     host,
+		"certPath": certPath,
+		"keyPath":  keyPath,
+	}).Debug("generating self-cert")
+	if err := CreateSelfCert(ctx, certPath, keyPath, host); err != nil {
```

Comment:
Hi @vladimirvivien,

Please relocate this code to the installer and do not generate certs unless the RR binary is a standard or controller binary.